### DISCUSSION TITLE
Code convention

### DISCUSSION THREAD

**Discussion by wlstmd1004v** [lion-icandoit/Icandoit-team13#10] [General]
code convention을 어떤식으로 작성해야 될지 정해야 될 것 같아요 dl,dt,dd 로 구현할지 아니면 ul li 로 구현하실지 정해서 코딩해야될 것 같습니다.

**Comment by jay0425** [comment]
저는 사진을 클릭 했을 때 어디론가 이동이 되는 것 때문에 ul - li - a - img 방식이 좋지 않을까 생각합니다

**Reply by wlstmd1004v** [reply]
저는 dl 밖에 li를 사용하고 a 태그 사용해서 img를 가지고 올 생각이었습니다.

**Reply by wlstmd1004v** [reply]
드라마 설명이기 떄문에 dl dt dd 구조로 구현해야된다고 생각합니다
